virt/virt-4-11-release-notes.adoc

```diff
@@ -105,6 +105,13 @@ Deprecated features are included in the current release and supported. However,
 Removed features are not supported in the current release.
 
 //CNV-16317 NMState is no longer part of CNV
+* {VirtProductName} {VirtVersion} removes support for the link:https://nmstate.io/[nmstate] library including the following objects:
++
+** `NodeNetworkState`
+** `NodeNetworkConfigurationPolicy`
+** `NodeNetworkConfigurationEnactment`
++
+To continue using `nmstate` features, you must first install the xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator] from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`), before updating to {VirtProductName} {VirtVersion}.
```

Comment:
Can we add that "All existing configuration will be preserved and will remain supported by the standalone Kubernetes NMState  Operator."